stats.php

```diff
@@ -109,6 +117,85 @@ function track_file_delete() {
 	] );
 }
 
+/**
+ * Tracks the authentication type used during successful XML-RPC requests.
+ */
+function determine_xmlrpc_password_type( $user, $username, $password ) {
```

Comment:
I wanted to test this behavior and I started digging into the WordPress core logic for the [xmlrpc-server](https://github.com/WordPress/WordPress/blob/3906e3e9d6d25225b41cda97f8e6f89f24e31211/wp-includes/class-wp-xmlrpc-server.php) to see how the authentication was handled.

One thing I noticed when testing this is that `WP_Application_Passwords::is_in_use()` will return `true`  even if we are using the user password.
This is caused by the `determine_xmlrpc_password_type`, I suspect some of the functions we call set this.

I don't think we want that that logic to change, we should find a way to introduce this tracking without changing any behavior.